### ISSUE TITLE
Metadata location

### ISSUE THREAD

**Issue by yaxu** [tidalcycles/Clean-Samples#10]
.cleanmeta looks like an unused extension: https://github.com/search?q=extension%3Acleanmeta&type=Code

but is there a better, more self-explanatory name we could use?

**Comment by charlieroberts** [comment]
perhaps `.samplemeta` is better? also not taken... 

another option might be for each repo to have a `meta` directory where the meta files are placed, and then just use the .json extension. That makes more sense to me personally... and no one would have specify JSON syntax highlighting in their editors for `.cleanmeta` (or other alternatives)

**Comment by telephon** [comment]
> another option might be for each repo to have a `meta` directory where the meta files are placed, and then just use the .json extension.

yes, that seems the best to me.

**Comment by yaxu** [comment]
It would be harder to find `meta/foo.json` files though. We could call the folder `_cleanmeta` ?

**Comment by yaxu** [comment]
Looking at this, it's a bit messy:
  https://github.com/tidalcycles/sounds-tr808-fischer

I agree it's better to put the metadata files in a subfolder. They could go in the same subfolder as the files they're about, but it's easier for superdirt and friends to discover them in a dedicated top-level subfolder. We could then say that `/foo` is the same as `../foo`, relative to the metadata file.

**Comment by yaxu** [comment]
Ok how about this as an example?
  https://github.com/tidalcycles/sounds-dirty/

I.e. as suggested above - .json files, in a _soundmeta folder, with sample paths in the metadata relative to the folder above _soundmeta.

I'm not completely sold on prefixing with `_`. I thought about making them hidden with `.soundmeta` but I couldn't think of a reason to do that.

**Comment by yaxu** [comment]
Ok I updated the script and used it to regenerate the metadata on the tr808 repo, e.g.: https://github.com/tidalcycles/sounds-tr808-fischer/blob/main/_soundmeta/cb8.json

There is a new 'sound_type' field, which is currently either 'sample', 'supercollider', or 'mixed' (for both). I thought this would be useful for looking for either scsynths or samples, or in e.g. gibber's case, ignoring the former

Makes sense?

**Comment by ndr-brt** [comment]
What about call the field just "type"? I think the "sound" prefix could be implicit.

**Comment by yaxu** [comment]
@ndr-brt Yes true, then we will have permission to use it for sharing other resources - videos, images, vector files etc.

Although 'resource' doesn't feel right, as that implies scarcity, and I think what we're doing is more about the commons and plenty, i.e. love. With all the nonsense around NFTs maybe this is important..
